radish34/zkp/package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "build": "babel src -d dist",
     "start": "npm run build && node ./dist/index.js",
-    "dev": "nodemon --exec babel-node ./src/index.js",
+    "dev": "npm run build && nodemon --exec babel-node ./src/index.js",
```

Comment:
might want to add the babel regenerator plugins as in: https://github.com/ethereum-oasis/baseline/commit/fe657277f4a58e8a06f13f6e6a406a23cc65ac48. this is to avoid any regeneration run time errors which pop otherwise

Reply:
https://github.com/babel/babel/issues/5085
https://github.com/babel/babel/issues/8829

Reply:
Added babel plugin to `api` and `zkp` services